### PR TITLE
Upcase country to match database format

### PR DESCRIPTION
**Pull request by patrickvinograd**
Fixes https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1805

### CODE REVIEW

File: spec/factories/institutions.rb [hunk 1]
```diff
@@ -11,42 +11,42 @@
     version 1
 
     trait :in_nyc do
-      city 'new york'
-      state 'ny'
-      country 'usa'
+      city 'NEW YORK'
```
Comment (patrickvinograd) [inline spec/factories/institutions.rb]: Updated factory to make data look like what is in DB.

File: spec/controllers/v0/institutions_controller_spec.rb [hunk 2]
```diff
@@ -56,6 +56,34 @@
...
+    end
+
+    it 'filter by lowercase country returns results' do
+      get :index, name: 'chicago', country: 'usa', version: 1
+      expect(JSON.parse(response.body)['data'].count).to eq(1)
+      expect(response.content_type).to eq('application/json')
+      expect(response).to match_response_schema('institutions')
+    end
+
+    it 'filter by uppercase state returns results' do
```
Comment (patrickvinograd) [inline spec/controllers/v0/institutions_controller_spec.rb]: state was actually working ok, now have specs to ensure that's the case.

File: app/controllers/v0/institutions_controller.rb [hunk 3]
```diff
@@ -45,9 +45,11 @@ def normalized_query_params
       query.tap do
         query[:name].try(:strip!)
         query[:name].try(:downcase!)
-        query[:state].try(:upcase!)
+        %i(state country).each do |k|
+          query[k].try(:upcase!)
+        end
         %i(type_name student_veteran_group yellow_ribbon_scholarship principles_of_excellence
```
Comment (patrickvinograd) [inline app/controllers/v0/institutions_controller.rb]: type_name is handled specially "school/employer" mapped to "not equal/equal OJT". All other filter fields here are booleans.

**Review by mphprogrammer — APPROVED** [review]
LGTM

**Comment by saneshark** [comment]
This is good for now, but i think we need investigate doing case insensitive collation on these table columns in a future iteration.

There is also Postgres `ILIKE`: more on that here: https://www.postgresql.org/docs/8.3/static/functions-matching.html

**Pull request merged**